### DISCUSSION TITLE
[SUGGESTION] Arena queue send msg only players in queue

### DISCUSSION THREAD

**Discussion by solidmaxtor** [azerothcore/azerothcore-wotlk#3935] [Ideas]
Its over Arena Queue. I am looking for where the order is given so that messages from id `718` and `719` of the `trinity_string` table are sent.
These messages are sent to the entire server when a team enters or leaves the arena.
I want to modify it to send it through a specific channel or show itself only to those who are in queue of arena that would be ideal, in the same way that happens with the queue of BG:
```
Battleground.QueueAnnouncer.Enable = 1
Battleground.QueueAnnouncer.PlayerOnly = 1
```
in arena case is only:
```
Arena.QueueAnnouncer.Enable = 1 (global send)
```

Now if this is active is VERY LONG SPAM

**Comment by FrancescoBorzi** [comment]
FYI: edited for better readability:

![image](https://user-images.githubusercontent.com/75517/59919700-6ecc5b80-9428-11e9-82cf-f87b4011f6a2.png)

**Comment by wizzymore** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/2299

Now we are using anti-spam for queue announcer

If you have `Arena.QueueAnnouncer.Enable = 1` it will send out announces globally, if you have `Arena.QueueAnnouncer.Enable = 0` it will send out announces only for the player that queues, don't think there is a need for both..
